src/config.ts

```diff
@@ -98,15 +98,29 @@ function interpolateSecrets(config: unknown, secrets: Secrets): GoResult<unknown
   return goSync(() => JSON.parse(interpolatedConfig.replace(ESCAPED_ES_MATCH_REGEXP, '$1')));
 }
 
-const loadConfig = (configPath = './config/config.json') => {
+const replaceInterpolationStrings = (obj: any): any => {
+  if (typeof obj === 'string') {
+    obj = obj.replace(/\${MNEMONIC}/g, 'test test test test test test test test test test test junk');
+    obj = obj.replace(/\${FUNDER_RPC_URL_(.*?)_1}/g, 'https://rpc.example.com');
```

Comment:
I think `/\${FUNDER_RPC_URL_(.*?)}/g` will be more safe and also eligible to capture multiple RPC URL for chain like FUNDER_RPC_URL_ETHEREUM_2 or FUNDER_RPC_URL_LOCALHOST etc.

Reply:
Good catch 👍🏻